src/Conversions.cc

```diff
@@ -24,6 +24,7 @@
 #include <ignition/msgs/ellipsoidgeom.pb.h>
 #include <ignition/msgs/geometry.pb.h>
 #include <ignition/msgs/gui.pb.h>
+#include <ignition/msgs/gps_sensor.pb.h>
```

Comment:
```suggestion
#include <ignition/msgs/gps_sensor.pb.h>
#include <ignition/msgs/gui.pb.h>
```
nit: alphabetical order